test/parallel/test-domain-multiple-errors.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
The only difference is that one of the tests emits the errors on the same domain object, because that was requested here. I’ve added comments referencing the other test file in each of them.